lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
```

Comment:
This was here to prevent people from trying to run the webpacker tasks (which don't work when run from manageiq/).

But I guess they'll get a nice warning about missing binstubs, so.. :)